docusaurus.config.js

```diff
@@ -21,10 +21,19 @@ const config = {
   },
   plugins: [
     "./src/plugins/fathom",
-    // "./src/plugins/vercel",
     "./src/plugins/koala",
     "./src/plugins/hubspot",
-    "@docusaurus/theme-mermaid",
+    [
+      "@docusaurus/theme-mermaid",
```

Comment:
@ntotten - Not sure if you have a preference of using this or not, but I need to ensure that our early betas with a link to /local-development-preview will be re-directed

Reply:
Use this instead: https://github.com/zuplo/docs/blob/f1784053dabd07a650a66be7f4739f625601cc70/vercel.json#L4

Reply:
Sounds good